_includes/v23.1/backups/backup-options-for-schedules.md

```diff
@@ -3,5 +3,6 @@
 `detached`<a name="detached"></a>                           | [`BOOL`](bool.html) / None                   | **Note:** Backups running on a schedule have the `detached` option applied implicitly. Therefore, you cannot modify this option for scheduled backups. <br><br>When a backup runs in `detached` mode, it will execute asynchronously. The job ID will be returned after the backup [job creation](backup-architecture.html#job-creation-phase) completes. Note that with `detached` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](backup.html#run-a-backup-asynchronously). To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement.
-`kms`                                                            | [`STRING`](string.html) |  The [key management service (KMS) URI](take-and-restore-encrypted-backups.html#uri-formats) (or a [comma-separated list of URIs](take-and-restore-encrypted-backups.html#take-a-backup-with-multi-region-encryption)) used to encrypt the files (`BACKUP` manifest and data files) that the `BACKUP` statement generates. This same KMS URI is needed to decrypt the file when it is used to [restore](take-and-restore-encrypted-backups.html#restore-from-an-encrypted-amazon-s3-backup) and to list the contents of the backup when using [`SHOW BACKUP`](show-backup.html). <br/><br/>Currently, AWS KMS and Google Cloud KMS are supported.
+<span class="version-tag">New in v23.1:</span> `EXECUTION LOCALITY` | Key-value pairs | Restricts the execution of the backup to nodes that match the defined locality filter requirements, e.g., `WITH EXECUTION LOCALITY = 'region=us-west-1a,cloud=aws'`. <br><br>See [Execution Locality Backups](execution-locality-backups.html) for usage and reference detail.
+`kms`                                                            | [`STRING`](string.html) |  The [key management service (KMS) URI](take-and-restore-encrypted-backups.html#uri-formats) (or a [comma-separated list of URIs](take-and-restore-encrypted-backups.html#take-a-backup-with-multi-region-encryption)) used to encrypt the files (`BACKUP` manifest and data files) that the `BACKUP` statement generates. This same KMS URI is needed to decrypt the file when it is used to [restore](take-and-restore-encrypted-backups.html#restore-from-an-encrypted-amazon-s3-backup) and to list the contents of the backup when using [`SHOW BACKUP`](show-backup.html). <br/><br/>AWS KMS, Azure KMS, and Google Cloud KMS are supported.
```

Comment:
```suggestion
`kms`                                                            | [`STRING`](string.html) |  The URI of the cryptographic key stored in a key management service (KMS), or a comma-separated list of key URIs, used to [take and restore encrypted backups with multi-region encryption](take-and-restore-encrypted-backups.html#take-a-backup-with-multi-region-encryption). Refer to [URI Formats](take-and-restore-encrypted-backups.html#uri-formats). The key or keys are  used to encrypt the manifest and data files that the `BACKUP` statement generates and to decrypt them during a [restore](take-and-restore-encrypted-backups.html) operation and to list the contents of the backup when using [`SHOW BACKUP`](show-backup.html). <br/><br/>AWS KMS, Google Cloud KMS, and Azure Key Vault are supported.
```
I think the link for restore is too specific to AWS so I removed the anchor. Also, aren't the key URIs used to encrypt and decrypt all backups, not just multi-regional ones? Feel free to ignore this if my edit makes no sense.